lib/matterhorn/error_handling.rb

```diff
@@ -4,6 +4,7 @@ module ErrorHandling
 
     included do
       rescue_from "ActionController::ActionControllerError", with: :handle_controller_error
+      rescue_from "Matterhorn::Ordering::InvalidOrder", with: :handle_controller_error
```

Comment:
Hmm, I think we should only rescue errors that we expect an api to ship with.  Sinc ethis error woudl be more for the API developers, I would just allow Matterhorn::Ordering::InvalidOrder to bubble up without any extra logic.